docs/changes/3.0.0.rst

```diff
@@ -1,6 +1,26 @@
 Changelog for Falcon 3.0.0
 ==========================
 
+
+Summary
+-------
+
+We are pleased to present Falcon 3.0, a major new release that includes :class:`ASGI-based <falcon.asgi.App>` :mod:`asyncio` and :class:`WebSocket <falcon.asgi.WebSocket>` support, fantastic :ref:`multipart/form-data parsing <multipart>`, better error handling, enhancements to existing features, and the usual assortment of bug fixes.
+
+This is easily the biggest release—in terms of both hours volunteered and code contributed—that we have ever done. We sincerely thank our stupendous group of 37 contributors who submitted pull requests for this release, as well as all those who have generously provided financial support to the project.
+
+When we began working on this release, we knew we wanted to not only evolve the framework's existing features, but also to deliver first-class, user-friendly :mod:`asyncio` support alongside our existing :class:`WSGI <falcon.App>` feature set.
+
+On the other hand, we have always fought the temptation to expand Falcon's scope, in order to leave room for community projects and standards to innovate around common, self-contained capabilities. And so when `ASGI <https://asgi.readthedocs.io/en/latest/>`_ arrived on the scene, we saw it as the perfect opportunity to deliver long-requested :mod:`asyncio` and :class:`WebSocket <falcon.asgi.WebSocket>` features while still encouraging sharing and reuse within the Python web community.
+
+It can be painful to migrate a large code base to a major new version of a framework. Therefore, in 3.0 we went to great lengths to minimize breaking changes, although a number of methods and attributes were deprecated. That being said, everyone will likely run up against at least one or two items in the breaking changes list below. Please carefully review the list of changes and thoroughly test your apps with Falcon 3.0 before deploying to production.
+
+Leading up to this release, members of the core maintainers spent many hours (and not a few late nights and weekends) prototyping, tuning, and testing in order to uphold the high standards of correctness and reliability for which Falcon is known. That being said, no code is perfect, so please don't hesitate to reach out on `falconry/user
```

Comment:
members of the core maintainers _team_ ?

Reply:
fixed